app/views/idv/in_person/address.html.erb

```diff
@@ -37,6 +37,8 @@
     <% end %>
   <%= render ValidatedFieldComponent.new(
         form: f,
+        hint: t('in_person_proofing.form.state_id.address1_hint'),
+        hint_class: 'puerto-rico-extras',
```

Comment:
You might need to rebase or merge the latest main, since `hint_class: '...'` was removed in favor of `hint_html: { class: '...' }` in #8232.

Reply:
i notice that the hint is flashing on the screen when the page is loaded. Not sure how to rectify this.

Reply:
I think you might have fixed it with 6f6263f ? Normally that might introduce other problems with supporting browsers where JavaScript is disabled, but since IdV requires JavaScript, it should be fine. An alternative might be to add the class server-side depending if the known address state is Puerto Rico.

Reply:
yes that commit fixed it, yay! I do need to test the remote flow before merging though